docker-for-mac/space.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Disk space
+keywords: mac, disk
+title: Disk space in Docker for Mac
+---
+
+Docker for Mac stores Linux containers and images in a single large "disk image" file
+in the Mac filesystem. This is different to Docker on Linux which usually stores containers
+and images in the `/var/lib/docker` directory.
+
+## Where is the "disk image" file?
+
+To locate the "disk image" file, first click on the whale menu and then click on
+"Preferences...". When the preferences window appears, click on the "Disk" icon
+and then "Reveal in Finder":
+
+![Disk preferences](images/settings-disk.png)
+
+The preferences window also shows how much disk space the file is currently consuming.
+In this example it is consuming only 2.4 GB out of a maximum of 64 GB.
+
+Note that other tools may display the space usage of the file incorrectly.
+
+## If the file is too big
```

Comment:
Should we mention the actual vs virtual size of the file?

Reply:
Yes good idea. It's mentioned further down but it would be good to bring it nearer the top since it's bound to be a frequent source of confusion.

Reply:
@thaJeztah @djs55 - Thank you both for the input! Please see my latest updates to see if this is what you are looking for and if more is needed.

Reply:
This looks fine to me!